### PR TITLE
Apis 1015 stop auto deploy

### PR DESCRIPTION
**Pull request by AndyBowes-EqualExperts**

### CODE REVIEW

File: app/connectors/ApiPublisherConnector.scala [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    http.GET[Seq[APIApprovalSummary]](s"$serviceBaseUrl/unapproved")
+      .recover {
+        case e: Upstream5xxResponse => throw new FetchApiDefinitionsFailed
+      }
+  }
+
+  def fetchApprovalSummary(serviceName: String)(implicit hc: HeaderCarrier) : Future[APIApprovalSummary] = {
+    http.GET[APIApprovalSummary](s"$serviceBaseUrl/$serviceName/summary")
+      .recover {
+        case e: Upstream5xxResponse => throw new FetchApiDefinitionsFailed
```
Comment (RawToast) [inline app/connectors/ApiPublisherConnector.scala]: This logic:

`.recover {
 +        case e: Upstream5xxResponse => throw new FetchApiDefinitionsFailed
 +      }`

Is used 2/3 times, so could be moved into a method. Moreover, the match is not safe as it is incomplete. The pattern match in approveService is better :)

File: app/controllers/DeploymentApprovalController.scala [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  override def authConnector = AuthConnector
+  override def authProvider = GatekeeperAuthProvider
+  override val appConfig = AppConfig
+}
+
+trait DeploymentApprovalController extends FrontendController with GatekeeperAuthWrapper {
+  val deploymentApprovalService: DeploymentApprovalService
+  implicit val appConfig: AppConfig
+
+  private def redirectToPendingPage() = {
```
Comment (RawToast) [inline app/controllers/DeploymentApprovalController.scala]: I don't like private/protected methods being near the top of a file, could you move the private/protected methods to one of:

* The bottom of the trait
* Inside a public method (if that is the only place the method is used)

**Comment by rm-hull** [comment]
This needs to be a separate microsite, not inside this repo

File: app/views/deploymentApproval/deploymentReview.scala.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
+@import model.APIApprovalSummary
+
+@(form: Form[controllers.HandleApprovalForm], apiDefinition: APIApprovalSummary)(implicit request: Request[_], userFullName: Option[String], flash: Flash, messages: Messages, appConfig: config.AppConfig)
```
Comment (RawToast) [inline app/views/deploymentApproval/deploymentReview.scala.html]: Could you add a unit test for the two new views, there should be a good example of how to do this.

**Comment by RawToast** [comment]
I agree with @rm-hull, this should be a contained in a separate service.

**Pull request closed without merging**